databaseutil/databaseutil.go

```diff
@@ -36,24 +36,21 @@ func InsertIntoUsersTable(
 	password []byte,
 	creationTime time.Time,
 ) error {
-	var row *sql.Row
-	{
-		sqlQuery := `
-			INSERT INTO users (display_name, email_address, password, creation_time)
-			VALUES ($1, $2, $3, $4)`
+	sqlQuery := `
+		INSERT INTO users (display_name, email_address, password, creation_time)
+		VALUES ($1, $2, $3, $4)`
 
-		row = db.QueryRow(sqlQuery, displayName, emailAddress, password, creationTime)
+	rows, err := db.Query(sqlQuery, displayName, emailAddress, password, creationTime)
+	if err != nil {
+		return convertPostgresError(err)
 	}
+	defer rows.Close()
 
-	if err := row.Scan(); err != nil {
-		if err == sql.ErrNoRows { // success
-			return nil
-		}
-
+	if err = rows.Err(); err != nil {
```

Comment:
random nitpicky thing, but we should do `:=` here. Honestly wish `err` wasn't in scope here, so this couldn't happen, but that doesn't seem worth the hassle.